src/styles.scss

```diff
@@ -1,22 +1,15 @@
-@use '~@angular/material' as mat;
+@use '@angular/material' as mat;
 @use 'theme';
 
 // Include the common styles for Angular Material. We include this here so that you only
 // have to load a single css file for Angular Material in your app.
 // Be sure that you only ever include this mixin once!
-@include mat.core(theme.$custom-typography);
+@include mat.core();
 
 // Include theme styles for core and each component used in your app.
 // Alternatively, you can import and @include the theme mixins for each component
 // that you are using.
-@include mat.core-theme(theme.$app-theme);
-@include mat.button-theme(theme.$app-theme);
-@include mat.icon-theme(theme.$app-theme);
-@include mat.toolbar-theme(theme.$app-theme);
-@include mat.divider-theme(theme.$app-theme);
-@include mat.expansion-theme(theme.$app-theme);
-@include mat.progress-spinner-theme(theme.$app-theme);
-@include mat.tabs-theme(theme.$app-theme);
+@include mat.all-component-themes(theme.$app-theme);
```

Comment:
Does this increase the CSS bundle size? (If so, how much?)

Reply:
Yes, calling this mixin increases the size of the bundle. Here are the details:
Within:
<img width="315" alt="image" src="https://user-images.githubusercontent.com/5073455/223484425-e5a173a3-2c5e-45d1-aba1-becba0a1d923.png">

Without:
<img width="310" alt="image" src="https://user-images.githubusercontent.com/5073455/223484468-8f82de6a-0581-48d6-b023-80a3dea2a193.png">

*127kB difference*

We can import thems only for components that are used in the application. What do you think about this @gkalpak?

Reply:
Both solutions are described. When the application does not use too many components, it is written that `mat.all-component-themes` will produce unnecessary CSS.

> As an alternative to listing every component that your application uses, Angular Material offers Sass mixins that include styles for all components in the library: all-component-colors, all-component-typographies, all-component-densities, and all-component-themes. These mixins behave the same as individual component mixins, except they emit styles for core-theme and all 35+ components in Angular Material. Unless your application uses every single component, this will produce unnecessary CSS.

I'm fine with both solutions in the demo app. I would have the same concerns as you if that would be a real-world app 😄

Reply:
As we discussed at the meeting, we leave the version `all-component-themes`